code/modules/atmospherics/machinery/components/components_base.dm

```diff
@@ -9,14 +9,14 @@
 	var/showpipe = TRUE
 	///When the component is on a non default layer should we shift everything? Or just the underlay pipe
 	var/shift_underlay_only = TRUE
-	///Stores the component pipeline
-	var/list/datum/pipeline/parents
 	///If this is queued for a rebuild this var signifies whether parents should be updated after it's done
 	var/update_parents_after_rebuild = FALSE
-	///Stores the component gas mixture
-	var/list/datum/gas_mixture/airs
 	///Handles whether the custom reconcilation handling should be used
 	var/custom_reconcilation = FALSE
+	///Stores the parent pipeline, used in components
+	var/list/datum/pipeline/parents
+	///Stores the gasmix for each node, used in components
+	var/list/datum/gas_mixture/airs
```

Comment:
and these

Reply:
Same

Reply:
These I'd prefer moved back, since the order appears to have been intentional

Reply:
Ehhh I really really doubt that it was done on purpose, but I'll move both up